mono/mini/method-to-ir.c

```diff
@@ -7701,18 +7701,25 @@ mono_method_to_ir (MonoCompile *cfg, MonoMethod *method, MonoBasicBlock *start_b
 
 		if ((cfg->method == method) && cfg->coverage_info) {
 			guint32 cil_offset = ip - header->code;
+			gpointer counter = &cfg->coverage_info->data [cil_offset].count;
```

Comment:
I think it should be possible. The only concern would be interaction with inlining, I guess (see the `cfg->method == method` check above).